docker/scripts/mingw_w64_float_fix.patch

```diff
@@ -0,0 +1,25 @@
+commit 82b169c5734a6198d3b4c51a48f82e7b7104f143
```

Comment:
Can you copy the contents of https://github.com/msys2-contrib/mingw-w64/blob/master/COPYING at the top of this file?

Reply:
@eustas  ping?